assets/javascript/app.js

```diff
@@ -99,35 +100,74 @@ function getGames(){
 function createCards(input){
```

Comment:
Lets look at adding the display results function in to this function, in order to pass that array of returned content.